openbb_platform/extensions/crypto/openbb_crypto/price/price_router.py

```diff
@@ -56,3 +60,16 @@ async def historical(
 ) -> OBBject:
     """Get historical price data for cryptocurrency pair(s) within a provider."""
     return await OBBject.from_query(Query(**locals()))
+
+
+@router.stream(methods=["GET"])
+async def live(symbol: str = "ethbtc", lifetime: int = 10, tld: str = "us") -> OBBject:
+    """Connect to Binance WebSocket Crypto Price data feed."""
```

Comment:
This endpoint will not generate any documentation or descriptions because it is not using the ProviderInterface.